resources/geometry.rb

```diff
@@ -517,48 +507,43 @@ def self.process_occupants(model, runner, num_occ, occ_gain, sens_frac, lat_frac
     people_sch = nil
     activity_sch = nil
 
-    # Get spaces
-    ffa_spaces = self.get_conditioned_spaces(model.getSpaces)
-
-    ffa_spaces.each do |space|
-      space_obj_name = "#{Constants.ObjectNameOccupants}|#{space.name.to_s}"
-
-      space_num_occ = num_occ * UnitConversions.convert(space.floorArea, "m^2", "ft^2") / cfa
+    space_obj_name = "#{Constants.ObjectNameOccupants}|#{space.name.to_s}"
 
-      if people_sch.nil?
-        # Create schedule
-        people_sch = MonthWeekdayWeekendSchedule.new(model, runner, Constants.ObjectNameOccupants + " schedule", weekday_sch, weekend_sch, monthly_sch, mult_weekday = 1.0, mult_weekend = 1.0, normalize_values = true, create_sch_object = true, schedule_type_limits_name = Constants.ScheduleTypeLimitsFraction)
-        if not people_sch.validated?
-          return false
-        end
-      end
+    space_num_occ = num_occ * UnitConversions.convert(space.floorArea, "m^2", "ft^2") / cfa
 
-      if activity_sch.nil?
-        # Create schedule
-        activity_sch = OpenStudio::Model::ScheduleRuleset.new(model, activity_per_person)
+    if people_sch.nil?
```

Comment:
No longer need `people_sch = nil` above and this line since we're no longer looping. Just create the schedule, period.